core/domain/exp_domain_test.py

```diff
@@ -686,9 +686,7 @@ def test_hints_validation(self):
         init_state.update_interaction_hints([{
             'hint_text': 'hint one',
         }])
-        self._assert_validation_error(
```

Comment:
Since we're preserving the old implementation for now, it would be best if we kept this in. Once we do the complete switch, this should be part of the checklist for final implementation.